docs/fsharp/using-fsharp-on-azure/queue-storage.md

```diff
@@ -124,40 +116,40 @@ software failure, another instance of your code can get the same message
 and try again. Your code calls `DeleteMessage` right after the message
 has been processed.
 
-[!code-fsharp[QueueStorage](~/samples/snippets/fsharp/azure/queue-storage.fsx#L75-L76)]
+[!code-fsharp[QueueStorage](../../../samples/snippets/fsharp/azure/queue-storage.fsx#L60-L61)]
 
 ## Use Async workflows with common Queue storage APIs
 
 This example shows how to use an async workflow with common Queue storage APIs.
```

Comment:
We should add a comment that all of the Queue methods we've shown so far have `Async` alternatives if customers would prefer the more performant option that doesn't block.